client/my-sites/plugins/marketplace/marketplace-domain-upsell/styles.scss

```diff
@@ -0,0 +1,104 @@
+//@import '~@automattic/marketplace/styles/mixins';
+
+.marketplace-domain-upsell__root {
+	margin-left: 34px;
+	display: grid;
+	grid-template-rows: 40px 100%;
+	grid-template-columns: 60% 40%;
+
+	*,
+	*::before,
+	*::after {
+		box-sizing: border-box;
+	}
+	
+	.marketplace-domain-upsell__shopping-cart-container {
+		padding-left: 50px;
+		max-width: 400px;
+	}
+
+	.marketplace-domain-upsell__header {
+		margin: 0 0 20px;
+		@media ( min-width: 782px ) {
+			margin: 0 0 20px;
+		}
+		@media ( min-width: 600px ) {
+			margin: 0 0 20px;
+		}
+	}
+	.marketplace-title {
+		font-family: var( --p2-font-inter );
+		letter-spacing: 0.4px;
+		line-height: 28px;
+		font-weight: 600;
+		font-size: 1.5rem;
+		margin: 0 0 30px;
+	}
+
+	.marketplace-subtitle {
+		font-family: var( --p2-font-inter );
+		font-style: normal;
+		font-weight: normal;
+		font-size: 0.875rem;
+		line-height: 16px;
+		color: #2c3338;
+		margin: 0 0 30px;
+	}
+	.marketplace-domain-upsell__back-button {
+		button {
+			font-family: var( --p2-font-inter );
+			font-style: normal;
+			letter-spacing: 0.4px;
+			color: #1d2327;
```

Comment:
Using hardcoded custom colors is suspicious: let's use appropriate predefined colors (with CSS variables) that can be modified by a custom Calypso theme.

Reply:
Fixed by 0aa0786